src/Sarif.Multitool.Library/KustoCommand.cs

```diff
@@ -130,15 +130,24 @@ private void InitializeKustoClient()
                         onDemandValidated = (sbyte)dataReader.GetValue(GetIndex(dataReader, dataReaderIndex, "OnDemandValidated")) == 1;
                     }
 
-                    if (etlEntity == "Build" || etlEntity == "Release" || etlEntity == "BuildDefinition" || etlEntity == "ReleaseDefinition")
+                    if (etlEntity == "Build" || etlEntity == "BuildDefinition" ||
+                        etlEntity == "Release" || etlEntity == "ReleaseDefinition")
                     {
                         itemPath = itemPath.Replace("vsrm.visualstudio.com", "visualstudio.com");
                         itemPath = itemPath.Replace("_apis/build/Definitions/", "_build?definitionId=");
                         itemPath = itemPath.Replace("_apis/Release/definitions/", "_release?_a=releases&view=mine&definitionId=");
+                        itemPath = itemPath.Replace("_apis/wit/workItems/", "_workitems/edit/");
```

Comment:
>Replace [](http://example.com/codeflow?start=44&length=7)

why is this in two places?

why would etl entity not be work item but the path would include work item?

Reply:
forgot to exclude. Changin in a new pr.